net/node/transactionPool.go

```diff
@@ -46,7 +50,31 @@ func (txnPool *TXNPool) GetTxnPool(cleanPool bool) map[common.Uint256]*transacti
 	if cleanPool == true {
 		txnPool.init()
 	}
-	return list
+	return DeepCopy(list)
+}
+
+func DeepCopy(mapIn map[common.Uint256]*transaction.Transaction) map[common.Uint256]*transaction.Transaction {
+	reply := make(map[common.Uint256]*transaction.Transaction)
+	for k, v := range mapIn {
+		reply[k] = v
+	}
+	return reply
+}
+
+// Attention: clean the trasaction Pool with committed transactions.
+func (txnPool *TXNPool) CleanTxnPool(txs []*transaction.Transaction) error {
+	txsNum := len(txs)
+	txInPoolNum := len(txnPool.list)
+	cleaned := 0
+	for _, tx := range txs {
+		delete(txnPool.list, tx.Hash())
+		cleaned++
+	}
+	if txsNum-cleaned != 1 {
```

Comment:
1.as the BookKeeping which all block has at the first transactions will not in the txnpool ,so there 2(txnpool & block.transactions) always have the value 1 difference . 
2. i have just runed the go fmt. but the result always be [txsNum-cleaned], there have no blank between '-' . could you check it again, thanks.